### PR TITLE
craft fix

### PR DESCRIPTION
**Pull request by PlasticCat**
## Description
fixed recipe to actually be craftable

## Motivation and Context
needed fixed

## How Has This Been Tested?
complied

## Changelog (necessary)
:cl:
tweak: fixed code
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/den.dm [hunk 1]
```diff
@@ -157,7 +141,7 @@ Farmer
 	total_positions = 4
 	spawn_positions = 4
 	supervisors = "the sheriff and the mayor"
-	description = "A local settler of Pahrump, you know a thing or two about cultivation and husbandry, making you a vital backbone of the town in growing medicines, crops and raising livestock in the service of the townsfolk."
+	description = "A local settler of Yuma, you know a thing or two about cultivation and husbandry, making you a vital backbone of the town in growing medicines, crops and raising livestock in the service of the townsfolk."
```
Comment (Laus4Deus) [inline code/modules/jobs/job_types/den.dm:144]: 😄

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 2]
```diff
@@ -575,3 +583,12 @@
 	filling_color = "#FFD700"
 	tastes = list("sickly fat" = 1)
 	foodtype = MEAT | JUNKFOOD | GROSS
+
+/obj/item/reagent_containers/food/snacks/crunchymutfruit	
+	name = "Crunchy Mutfruit"
+	desc = "Crunchy mutfruit is a fruit of a slight purple tinge that resembles an apple."
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/food/snacks_other.dm:589]: The description doesn't match the thing tho "A juicy mutfruit with a crispy exterior layer" or somthing

File: code/modules/food_and_drinks/food/snacks_vend.dm [hunk 3]
```diff
@@ -45,8 +42,7 @@
 /obj/item/reagent_containers/food/snacks/no_raisin
 	name = "4no raisins"
 	icon_state = "4no_raisins"
-	desc = "The best raisins in United States of American. Not sure why."
-	trash = /obj/item/trash/raisins
+	desc = "The best raisins in United States of American."
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/food/snacks_vend.dm:45]: United States of America**n** 
Change pls

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_wastefood.dm [hunk 4]
```diff
@@ -100,3 +97,24 @@
...
+    	/obj/item/reagent_containers/food/snacks/grown/mutfruit = 1,
+    	/obj/item/reagent_containers/food/snacks/butter = 1,
+    	/datum/reagent/consumable/sugar = 5
+	)
+	result = /obj/item/reagent_containers/food/snacks/crunchymutfruit	
+	category = CAT_FOOD
+	subcategory = CAT_WASTEFOOD
+
+/datum/crafting_recipe/food/tatofries
+	name = "crunchy mutfruit"
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_wastefood.dm:113]: isn't this an error?

**Comment by goodolrusty** [comment]
This PR will cause an overwrite of another PR, due to the wasteland.dm file being needed in PR 417 to overhaul waster loadouts.

File: code/modules/crafting/recipes.dm [hunk 5]
```diff
@@ -1893,7 +1834,7 @@
 	category = CAT_MISC
 	subcategory = CAT_MISC
 
-	/datum/crafting_recipe/SalvageCandle
+/datum/crafting_recipe/salvagecandle
```
Comment (Laus4Deus) [inline code/modules/crafting/recipes.dm:1837]: 👍

**Pull request merged**